Lesson27/ezgi/create/create-script.js

```diff
@@ -0,0 +1,46 @@
+// const urlParams = new URLSearchParams(window.location.search);
```

Comment:
No url parameters are available here so remove this and next line. It's a good practice to always clean up unused code.